ecc/bls12-377/fp/element_ops_amd64.go

```diff
@@ -41,3 +44,64 @@ func reduce(res *Element)
 //
 //go:noescape
 func Butterfly(a, b *Element)
+
+// Mul z = x * y (mod q)
+//
+// x and y must be strictly inferior to q
```

Comment:
This is a nitpick but I think it's more common to say "less than" than "inferior to".